core/field_angle.js

```diff
@@ -155,7 +155,8 @@ Blockly.FieldAngle.prototype.showEditor_ = function() {
   this.line_ = Blockly.utils.createSvgElement('line',
       {'x1': Blockly.FieldAngle.HALF,
       'y1': Blockly.FieldAngle.HALF,
-      'class': 'blocklyAngleLine'}, svg);
+      'class': 'blocklyAngleLine',
```

Comment:
blocklyAngleLine only refers to the line on the angle picker, so changing it in css.js is reasonable.

After making the changes, commit on the same branch locally, then push to your remote branch (K-ran:bugfix/rounding_error).  No need to make a new pull request.